package.json

```diff
@@ -69,6 +70,7 @@
     "eslint-plugin-react-hooks": "1.x",
     "fast-xml-parser": "^3.14.0",
     "gh-pages": "^2.1.1",
+    "https": "^1.0.0",
```

Comment:
Don't add this -- it's a 6 year old pkg. You're just requiring Node's built-in `http` module.